modules/ossm-config-operator-infrastructure-node.adoc

```diff
@@ -18,29 +18,36 @@ If the operator will run on a worker node, skip this task.
 
 .Procedure
 
+. List the operators installed in the namespace by running the following command at the CLI:
```

Comment:
Just a suggestion,  the end of the sentence could be omitted completely:
```suggestion
. List the operators installed in the namespace:
```

Reply:
done